run_dir/design/projects.html

```diff
@@ -273,6 +273,23 @@ <h4 class="search-action" data-action="filterHeader" id="undefined">Undefined <i
               </div>
             </div>
           </div>
+          <div class="form-group row row-eq-height">
+            <div class="col-md-12 panel-group">
+              <div class="panel panel-default">
+                <div class="panel-heading">
+                  <h4 class="panel-title">
+                    Reorder Table Fields
+                  </h4>
```

Comment:
Would be nice with a little instruction on how to reorder the table fields here?

Reply:
There is an existing 'Info' button in the 'Customise' panel, so do you think it would be better to add the help text there or have a new button/text at this location?